.github/workflows/release_js_project.yaml

```diff
@@ -42,3 +42,34 @@ jobs:
       branch: releases/v${{ needs.extractChangelog.outputs.version }}
       version: v${{ needs.extractChangelog.outputs.version }}
       body: ${{ needs.extractChangelog.outputs.body }}
+  updateMajorTag:
+     runs-on: ubuntu-latest
+     permissions:
+       contents: write
+     needs: [extractChangelog, createRelease]
```

Comment:
this should probably just depend on createBranch as well since it uses the branch

Reply:
yes i agree i will add it